### DISCUSSION TITLE
Installing webots 2022b in Ubuntu 22

### DISCUSSION THREAD

**Discussion by royito55** [cyberbotics/webots#5021] [General]
Hello, 

I just wanted to ask whether there are major changes in the way webots works from Ubuntu 20 to Ubuntu 22. 

I have an issue (in Ubuntu 22) with the downloading of textures when I install 2022a, which are solved by installing 2022b instead. Combine that with the new way to use the extern controllers (no `WEBOTS_SERVER` file), I want to go ahead and keep trying with this newer version.

Should I expect things to work? I also saw that there is a newer version 2023a, so I guess I'll be checking that one out too!

**Comment by stefaniapedrazzi** [comment]
Webots 2022b (will be released very soon)  is fully working on Ubuntu 22.04.
The only major change with respect to R2022a is the way how PROTO are declared in files but it should be quite straightforward to upgrade the simulations.

Webots 2022a is not working out of the box on Ubuntu 22.04.
In the documentation you can find some additional information about how to install it on for Ubuntu 22.04:
https://www.cyberbotics.com/doc/guide/installation-procedure#installing-the-debian-package-directly

Webots 2023a is still in development and we may introduce many changes so I would not recommend you to use it unless you really need any new of the new functionalities (currently none).

**Reply by royito55** [reply]
Thank you for your answer. I will then stick with 2022a in an Ubuntu 20 machine until 2022b is released.
